.goreleaser.yml

```diff
@@ -6,6 +6,12 @@ release:
   draft: true
   prerelease: auto
 
+before:
+  hooks:
+    - go mod download
+    - go install github.com/gobuffalo/packr/v2/packr2@v2.8.3
+    - packr2
```

Comment:
Just curious:
when and if we approach with the embedded migrations, we wouldn't need this part as the pre-execution step?

Reply:
that's right